### PR TITLE
Changes on tool readmes

### PR DESCRIPTION
**Pull request by sima-zhu**
More changes on Readme files.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,13 +14,16 @@ Add code owner per tool in [CODEOWNER](https://github.com/Azure/azure-sdk-tools/
 
 ## README
 
-Please add README.md file for every tool to illustrate:
+1. Please add README.md file for every tool to illustrate:
```
Comment (benbp) [inline CONTRIBUTING.md:17]: The way these numbered items render is probably not what you intend:

image

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -14,13 +14,16 @@ Add code owner per tool in [CODEOWNER](https://github.com/Azure/azure-sdk-tools/
...
 * The purpose of the tool.
 * Prerequisites before use
 * How to use, test and maintain the tool locally and remotely. 
 * Better to include where the tool is being used.
 
-Example: [README](https://github.com/Azure/azure-sdk-tools/blob/main/tools/http-fault-injector/README.md) 
+Example: [README.md](https://github.com/Azure/azure-sdk-tools/blob/main/tools/http-fault-injector/README.md) 
+
+2. Please add tool index of the tool in root [README.md](https://github.com/Azure/azure-sdk-tools/blob/main/README.md).
```
Comment (benbp) [inline CONTRIBUTING.md:25]: In my opinion, "please" sounds a little odd to use in technical documentation. No need for politeness in impersonal and imperative instructions.

Also, I'd direct link to the index and rephrase a little, as the instruction was confusing to me without having looked at the readme first:

```suggestion
2. Add tool details to the index in the root [README.md](https://github.com/Azure/azure-sdk-tools/blob/main/README.md).
https://github.com/Azure/azure-sdk-tools/blob/main/README.md#index
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -40,6 +43,10 @@ Example: [ci.yml](https://github.com/Azure/azure-sdk-tools/blob/main/tools/Creat
 
 - Pipelines should use common templates from `eng/pipelines/templates`.
 
+<<<<<<< HEAD
```
Comment (weshaggard) [inline CONTRIBUTING.md:46]: Bad merge here.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,39 +2,41 @@
...
 This is the repository used to host tools and libraries used by Azure SDK team engineering system. 
 We allow to create tools in any language suitable for the tools.
-For every newly developing tool, create a new folder with name of brief description of the tool. 
+For every newly developing tool, create a new folder with brief description on its name. 
 Do not add third party tools directly in your folder, use it as dependencies.
 
 ## Codeowners
 
-Add code owner per tool in [CODEOWNER](https://github.com/Azure/azure-sdk-tools/blob/main/.github/CODEOWNERS) with following format:
+Add code owner in [CODEOWNER](https://github.com/Azure/azure-sdk-tools/blob/main/.github/CODEOWNERS) with following format:
```
Comment (weshaggard) [inline CONTRIBUTING.md:10]:
```suggestion
Add code owner in [CODEOWNERS](https://github.com/Azure/azure-sdk-tools/blob/main/.github/CODEOWNERS) with following format:
```

**Review by weshaggard — APPROVED** [review]

**Pull request merged**